lighthouse-core/test/audits/dobetterweb/no-vulnerable-libraries-test.js

```diff
@@ -99,3 +100,41 @@ describe('Avoids front-end JavaScript libraries with known vulnerabilities', ()
     assert.equal(auditResult.score, 1);
   });
 });
+
+// https://github.com/npm/node-semver/issues/166#issuecomment-245990039
+function hasUpperBound(rangeString) {
+  const range = new semver.Range(rangeString);
+  if (!range) return false;
+
+  // For every subset ...
+  for (const subset of range.set) {
+    // Upperbound exists if...
+
+    // < or <= is in one of the subset's clauses (= gets normalized to >= and <).
+    if (subset.some(comparator => comparator.operator.match(/^</))) {
+      continue;
+    }
+
+    // Subset has a prerelease tag (operator will be empty).
+    if (subset.length === 1 && subset[0].operator === '') {
+      continue;
+    }
+
+    // No upperbound for this subset.
+    return false;
+  }
+
+  return true;
+}
+
+describe('every snyk vulnerability has an upper bound', () => {
```

Comment:
needs an `it()` or changed to `it` and nested in a `describe()` above, otherwise on failures you end up with

![Screen Shot 2019-07-10 at 11 52 18](https://user-images.githubusercontent.com/316891/60996338-4211b280-a309-11e9-9bd3-14303eba25e6.png)